src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -862,7 +862,7 @@ def add_service(
     def compile(
         self,
         model: typing.Optional[str] = None,
-        service_id: typing.Optional[uuid.UUID] = None,
+        service_id: typing.Optional[uuid.UUID | str | list[uuid.UUID | str]] = None,
```

Comment:
This makes it more user friendly, but type wise a bit messy.

It you oppose to this, I'm very willing to change this.